app/views/amends/ViewReturnSummaryView.scala.html

```diff
@@ -53,11 +55,19 @@ <h1 class="govuk-heading-xl">@returnPeriod</h1>
     <p>@link("view-returns", messages("viewReturnSummary.viewReturnsLink"), controllers.amends.routes.SubmittedReturnsController.onPageLoad())</p>
 
     @if(appConfig.isAmendsFeatureEnabled){
-        @amendCall.fold{
-            <h2 class="govuk-heading-m">@messages("viewReturnSummary.amend.notamendable.h2")</h2>
-            <p>@messages("viewReturnSummary.amend.notamendable.p1")</p>
-        }{ amendCall =>
-            <p>@link("amend-return", messages("viewReturnSummary.amendLink"), amendCall)</p>
+        @amendCall match {
+            case Right(call) => {
+                <p>@link("amend-return", messages("viewReturnSummary.amendLink"), call)</p>
+            }
+            case Left(DDInProgress) => {
+                <h2 class="govuk-heading-m">@messages("viewReturnSummary.amend.notamendable.h2")</h2>
+                <p>@messages("viewReturnSummary.amend.notamendable.p1")</p>
```

Comment:
not to self. this needs to use paragraphBody component as the free p tag css is gone.